pkg/services/featuremgmt/static_provider_test.go

```diff
@@ -56,3 +56,37 @@ func provider(t *testing.T, conf []byte) *OpenFeatureService {
 	require.NoError(t, err)
 	return p
 }
+
+func Test_CompareStaticProviderWithFeatureManager(t *testing.T) {
```

Comment:
Not sure if it is useful, just wanted to verify that enabled feature flags match between current service and new service.

Reply:
Let's keep this until we remove the legacy featuremanager!